.circleci/config.yml

```diff
@@ -14,13 +14,10 @@ defaults: &defaults
   - checkout
   - run:
       name: Install dependencies
-      command: pipenv install -r $REQUIREMENTS_FILE nose
-  - run:
-      name: Lint
-      command: pipenv run flake8
+      command: pip install --user -r $REQUIREMENTS_FILE nose flake8
```

Comment:
You probably did not intend to replace _running_ (`pipenv run`) flake8 with _installing_ it? It's already installed as part of `requirements.txt`.